torchvision/ops/feature_pyramid_network.py

```diff
@@ -45,15 +61,22 @@ class FeaturePyramidNetwork(nn.Module):
 
     def __init__(self, in_channels_list, out_channels, extra_blocks=None):
         super(FeaturePyramidNetwork, self).__init__()
-        self.inner_blocks = nn.ModuleList()
-        self.layer_blocks = nn.ModuleList()
+        inner_blocks = []
+        layer_blocks = []
         for in_channels in in_channels_list:
             if in_channels == 0:
                 continue
             inner_block_module = nn.Conv2d(in_channels, out_channels, 1)
             layer_block_module = nn.Conv2d(out_channels, out_channels, 3, padding=1)
-            self.inner_blocks.append(inner_block_module)
-            self.layer_blocks.append(layer_block_module)
+            inner_blocks.append(inner_block_module)
+            layer_blocks.append(layer_block_module)
+
+        self.layers = nn.ModuleList()
+        for inner, layer in zip(inner_blocks[:-1][::-1], layer_blocks[:-1][::-1]):
+            self.layers.append(FeaturePyramidBlock(inner, layer))
```

Comment:
FYI this unfortunately breaks backwards-compatibility with previously-serialized models, because now the state-dict is different.

Reply:
Is is possible to use `load_from_state_dict` to fix things up so it would work with the new names?

Reply:
Yes, it could be a possibility, adding a versioning there.